client/generated_update_snapshot_max_count_input.go

```diff
@@ -7,7 +7,7 @@ const (
 type UpdateSnapshotMaxCountInput struct {
 	Resource `yaml:"-"`
 
-	SnapshotMaxCount int `json:"snapshotMaxCount,omitempty" yaml:"snapshot_max_count,omitempty"`
+	SnapshotMaxCount int64 `json:"snapshotMaxCount,omitempty" yaml:"snapshot_max_count,omitempty"`
```

Comment:
It looks like go-rancher use `int64` instead of `int`. `NumberOfReplicas` also has similar behavior.

https://github.com/longhorn/longhorn-manager/blob/98189bfad382f206c0a6ba13e1dbde6283707795/k8s/pkg/apis/longhorn/v1beta2/volume.go#L277

https://github.com/longhorn/longhorn-manager/blob/98189bfad382f206c0a6ba13e1dbde6283707795/client/generated_volume.go#L60